src/app/admin/pages/reviewers/reviewers.component.ts

```diff
@@ -0,0 +1,190 @@
+import { Component, OnInit, OnDestroy, Output, EventEmitter, Input, IterableDiffers, IterableDiffer, DoCheck  } from '@angular/core';
```

Comment:
Is there a reason this component was copied over and slightly modified instead of made more generic and reused?